readme.md

```diff
@@ -110,6 +110,7 @@ Thanks for contributing! 🦋🙌
 - [](# "file-finder-buffer") [Lets you start typing your search immediately after invoking the File Finder (<kbd>t</kbd>), instead of having you wait for it to load first.](https://user-images.githubusercontent.com/1402241/75542106-1c811700-5a5a-11ea-8aa5-bea0472c59e2.gif)
 - [](# "preserve-file-finder-term") Preserves the search terms when navigating back and forth between the File Finder and the files.
 - [](# "show-associated-branch-prs-on-fork") [Shows the associated pull requests on branches for forked repositories.](https://user-images.githubusercontent.com/16872793/81504659-7e5ec800-92b8-11ea-9ee6-924110e8cca1.png)
+- [](# "feature-descriptions") [Shows the description and screenshot of each Refined GitHub feature when visiting the features’ file.](https://user-images.githubusercontent.com/16872793/97794235-b8f14f80-1bcd-11eb-9e66-de7c6880a162.gif)
```

Comment:
Since this feature is enabled exclusively on very specific files on our repo, it neither needs to be documented nor should it be disableable. Can you add an exception in `webpack.config.ts` so there are no errors and it doesn't appear in `options.html`?

Reply:
That also means that the feature won't be enabled on its own `.tsx` file :sweat_smile:

Reply:
Can't you just exclude it In the index.tsx and that would exclude it from the options?

Reply:
> That also means that the feature won't be enabled on its own `.tsx` file 😅

Sounds like something that can be fixed with… code. 👌

Reply:
> Since this feature is enabled exclusively on very specific files on our repo, it neither needs to be documented nor should it be disableable.